marketplace-client/marketplace-client-config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "modelPackage": "org.candlepin.subscriptions.marketplace.api.model",
+  "apiPackage": "org.candlepin.subscriptions.marketplace.api.resources",
+  "invokerPackage": "org.candlepin.subscriptions.marketplace",
+  "groupId": "org.candlepin",
+  "artifactId": "marketplace-client",
+  "java8": true
```

Comment:
Java 11 now!

Reply:
According to https://openapi-generator.tech/docs/generators/java/

> Use Java8 classes instead of third party equivalents. Starting in 5.x, JDK8 is the default and the support for JDK7, JDK6 has been dropped

so I'll remove this now unneeded config option :+1: